### ISSUE TITLE
Allow default directory of module_dir

### ISSUE THREAD

**Issue by matthelliwell2** [jasmine/IdeaJasmine#10]
When I am running my jasmine tests, I need to pass in node options "-r ts-node/register/transpile-only -r tsconfig-paths/register " as I'm running Typescript. For this to work I have to set the working directory to the module directory.

As I have many modules I have to manually set this all over the place. It would be much better if in run/edit configurations->templates->jasmine I could set the Working Directory to be $MODULE_DIR$ so that when a new run configuration is created it defaults to the correct module directory.

**Comment by slackersoft** [comment]
Based on my quick searching through the codebase, I would kind of expect this to work right now. We have tests that are checking that `$PROJECT_DIR$` seems to function correctly, and the plugin uses the IntelliJ `GeneralCommandLine` to set the working directory. Can you give me a bit more information on how exactly it seems to be failing?

**Comment by matthelliwell2** [comment]
I'm out if the office at the moment. When I'm back I'll reproduce it and
zip the project so you can have a look.

In the meantime, I think the issue is with multimodule projects. when you
add a second module to the project, it resolves some things relative to the
project director rather than the module sir.

**Comment by matthelliwell2** [comment]
I've reproduced the problem. I've attached a small project to replicate the issue.

The problem I have is that I'm running Typescript and using ts-node to transpile the ts files. ts-node is installed as a dependency in each module.

In run configurations->Templates->Jasmine the node options are set to "-r ts-node/register/transpile-only"

When the jasmine runner runs the test it executes a command like

"E:\Program Files\nodejs\node.exe" -r ts-node/register/transpile-only ....

It then gives an error

"Error: Cannot find module 'ts-node/register/transpile-only'"

If I change the working directory to be the module directory then it works.

A work around would be to install ts-node globally or provide an absolute path to ts-node. However ideally I need to be able to use the ts-node version installed in the module. 
[jasmineissue.zip](https://github.com/jasmine/IdeaJasmine/files/3097560/jasmineissue.zip)
